src/context/header-repository.ts

```diff
@@ -19,4 +19,15 @@ export class HeaderRepository extends BaseRepository<HeaderEntity> {
       .getRawOne();
     return height;
   }
+
+  public async getLastBlockId(): Promise<number | undefined> {
```

Comment:
Let's keep it semantic, please rename `getLastBlockId()` to `getLastHeaderId()`.